user-registration-app/src/app/views/live-view/live-view.component.html

```diff
@@ -711,6 +832,25 @@
                 </ul>
               </div>
             </div>
+            <div class="card intermediate">
+              <div class="card-image">
+                <img class="activator" src="../../../assets/images/workshop_leaders/BrianOuzomgi.jpeg">
+                <a class="waves-effect waves-light activator red btn">click to learn more</a>
+              </div>
+              <div class="card-content pt-5">
+                <span class="card-title activator text-darken-4" style="font-weight:bold;">
+                  Building and Selling a Great Product</span>
+                <div style="font-weight:bold;">Casey Collins, Brian McCombs, & Josh Kim</div>
```

Comment:
This name does not match the names listed in the card.

Reply:
Brian Ouzomgi's picture is used twice. We don't have access to any of the actual presenters pictures